src/components/getLocalFonts.ts

```diff
@@ -0,0 +1,35 @@
+import { Platform } from 'react-native'
+
+export const getLocalFonts = (): string => {
+  const fontUrlRegular = Platform.select({
+    ios: 'Lato-Regular.ttf',
+    android: 'file:///android_asset/fonts/Lato-Regular.ttf'
+  })
```

Comment:
nit: putting the nullish coalescing operator here instead of in the templated return would make it easier to read and would be more type safe.
 
```suggestion
  }) ?? ''
```

Reply:
Agree